hibernate-core/src/main/java/org/hibernate/persister/entity/AbstractEntityPersister.java

```diff
@@ -2728,6 +2729,49 @@ protected boolean hasWhere() {
 		return sqlWhereString != null;
 	}
 
+	private ModelPart findNestedSubPart(String attributeName) {
```

Comment:
I introduced a new method `findByPath` which you should be able to use instead of this.

Reply:
Nice! Fixed